docs/how-to/uc-image-creation.rst

```diff
@@ -0,0 +1,307 @@
+How to create a real-time Ubuntu Core image
+===========================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+Canonical builds and publishes Ubuntu Core images for a range of `supported platforms`_.
+However, these images include the generic Linux kernel. 
+In order to run Ubuntu Core with the real-time kernel, we need to build it ourselves. 
+
+This guide shows how to build an Ubuntu Core image with the real-time kernel.
+We start by showing how to :ref:`build a vanilla image <ubuntu-core-image>` which is excellent for testing and tuning, and then illustrate how to :ref:`make a custom image <custom-ubuntu-core-image>` with production-ready kernel configurations.
```

Comment:
Nitpick on the language here to be a bit more direct since this is a how-to.

```suggestion
We will first need to :ref:`build a vanilla image <ubuntu-core-image>` - which is excellent for testing and tuning - and proceed to :ref:`make a custom image <custom-ubuntu-core-image>` with production-ready kernel configurations.
```